src/main/java/org/isf/accounting/gui/BillDataLoader.java

```diff
@@ -0,0 +1,75 @@
+package org.isf.accounting.gui;
```

Comment:
Missing license header; it can by added automatically by running ` mvn license:format` in the root directory of the repo.

Reply:
solved with https://github.com/informatici/openhospital-gui/pull/220/commits/652c32a717b5185b83c06451a5457b9618a8b063